### PR TITLE
Readme change

### PR DESCRIPTION
**Pull request by thejohnjansen**
Adding a list of chapters to help track tests as they either migrate from old-tests, or if we create new tests. Putting this in 'old-tests' because that seems like a good spot to start when creating tests.

### CODE REVIEW

**Comment by sideshowbarker** [comment]
Shouldn’t this change be made to the https://github.com/w3c/web-platform-tests/blob/master/webdriver/README.md file instead? Otherwise, why we do still have webdriver tests in the `oldtests` directory? Is there a plan to move them to the top-level `webdriver` directory eventually?

**Comment by andreastt** [comment]
The intention is to use the tests in _/old-tests_ for inspiration. This change should go in _/webdriver/README.md_ as you say.

**Comment by thejohnjansen** [comment]
Updated to put this change into the webdriver directory instead of the old-tests directory.

File: webdriver/README.md [hunk 1]
```diff
@@ -1,8 +1,132 @@
...
-so that different driver implementations can determine
-whether they meet the recognised standard.
+Herein lies a set of conformance tests 
+for the W3C web browser automation specification 
+known as [WebDriver](http://w3c.github.io/webdriver/webdriver-spec.html). 
+The purpose of these tests is determine implementation compliance 
+so that different driver implementations can determine 
+whether they meet the recognised standard. 
+
+## How to run the tests
```
Comment (andreastt) [inline webdriver/README.md]: This whole section is wrong.

File: webdriver/README.md [hunk 2]
```diff
@@ -1,8 +1,132 @@
...
+   `WD_BROWSER=chrome python runtests.py`
+
+To run a specific test file you can just run `python test_file.py`
+
+Similarly you can specify a different browser to run against in webdriver.cfg:
+  `WD_BROWSER=chrome python ecmascript/ecmascript_test.py`
+
+Note: that you will likely need to start the driver's server before running.
+
+## Updating configuration
```
Comment (andreastt) [inline webdriver/README.md]: This should also be removed.

File: webdriver/README.md [hunk 3]
```diff
@@ -1,8 +1,132 @@
...
+
+The _webdriver.cfg_ file holds any configuration that the tests might
+require.  Change the value of browser to your needs.  This will then
+be picked up by WebDriverBaseTest when tests are run.
+
+Be sure not to commit your _webdriver.cfg_ changes when your create or modify tests.
+
+## How to write tests
```
Comment (andreastt) [inline webdriver/README.md]: This is also outdated.

File: old-tests/webdriver/README.md [hunk 4]
```diff
@@ -12,12 +12,12 @@ whether they meet the recognized standard.
 3. Run the test against a different config specified in webdriver.cfg:
    `WD_BROWSER=chrome python runtests.py`
 
-To be run a specific test file you can just run `python test_file.py`
+To run a specific test file you can just run `python test_file.py`
```
Comment (andreastt) [inline old-tests/webdriver/README.md]: Please revert changes to the file in old-tests.

**Comment by andreastt** [comment]
Please address code review comments and squash.

**Review by andreastt — CHANGES_REQUESTED** [review]

File: webdriver/README.md (hunk 2 above)
Comment (thejohnjansen) [inline webdriver/README.md]: Fixed

File: webdriver/README.md (hunk 3 above)
Comment (thejohnjansen) [inline webdriver/README.md]: Fixed

File: old-tests/webdriver/README.md (hunk 4 above)
Comment (thejohnjansen) [inline old-tests/webdriver/README.md]: Done, but for some reason the final line still shows as different. I ran the 'rich diff' and it said there was nothing to show. I'm not sure how to fix that.

File: webdriver/README.md (hunk 1 above)
Comment (thejohnjansen) [inline webdriver/README.md]: I didn't write that part, it was already there. Do you want me to remove it for this PR?

File: old-tests/webdriver/README.md [hunk 5]
```diff
@@ -35,4 +35,4 @@ Be sure not to commit your _webdriver.cfg_ changes when your create or modify te
    reused between tests. HTML files and other support files
    should be stored in a folder named 'res'.
 3. Test name should explain the intention of the test e.g. `def
-   test_navigate_and_return_title(self):`
+   test_navigate_and_return_title(self):`
```
Comment (andreastt) [inline old-tests/webdriver/README.md]: This should not change

File: old-tests/webdriver/README.md (hunk 4 above)
Comment (thejohnjansen) [inline old-tests/webdriver/README.md]: SWEET! Fixed. Thanks for that.

File: old-tests/webdriver/README.md (hunk 5 above)
Comment (thejohnjansen) [inline old-tests/webdriver/README.md]: Fixed

**Review by andreastt — APPROVED** [review]

**Pull request merged**